gcloud-java-datastore/src/main/java/com/google/gcloud/datastore/StructuredQuery.java

```diff
@@ -665,12 +674,18 @@ public B clearOrderBy() {
       return self();
     }
 
+    /*
+     * Clears all previously-specified OrderBy objects and orders by the given input instead.
```

Comment:
Alternative (pick what ever sounds better to you) could be
"Sets the query's order by clause (clearing any previously specified OrderBy settings)."